backend/Makefile

```diff
@@ -12,24 +12,37 @@
 OUT_DIR = dist/
 DATA_DIR = data
 
-BUILD_ARGS_BASE = -p src --distpath $(OUT_DIR) -y --clean -F
+################################### Building ###################################
+################## Unix and Windows use different delimiters ###################
+################################################################################ 
+ifeq ($(OS),Windows_NT)  # Windows_NT, XP, 2000, 7, 8, 10, 11...
+	DELIMITER = ;
+else
+	DELIMITER = :
+endif
+BUILD_ARGS = -p src --distpath $(OUT_DIR) -y --clean -F --add-data $(DATA_DIR)$(DELIMITER)$(DATA_DIR)
 
-# UNIX like systems use ':' as the separator while Windows uses ';'
-BUILD_ARGS_UNIX = $(BUILD_ARGS_BASE) --add-data $(DATA_DIR):$(DATA_DIR)
-BUILD_ARGS_WIN = $(BUILD_ARGS_BASE) --add-data $(DATA_DIR);$(DATA_DIR)
-
-make: unix
-
-unix:
-	pyinstaller $(BUILD_ARGS_UNIX) src/BeyondRGB.py
-
-win:
-	pyinstaller $(BUILD_ARGS_WIN) src/BeyondRGB.py
+make: 
+ifeq ($(OS),Windows_NT)  # Windows_NT, XP, 2000, 7, 8, 10, 11...
+	python.exe -m PyInstaller $(BUILD_ARGS) src/BeyondRGB.py
```

Comment:
for some unknown reason when i downloaded this to test it, it only worked with ```pyinstaller $(BUILD_ARGS) src/BeyondRGB.py``` so i guess you can dump the conditional here. other people testing on windows would be great

Reply:
Lets wait to get other tests before changing. We can get these at the meeting tommorow